### PR TITLE
Edited Readme File.

### PR DESCRIPTION
**Pull request by himanshuchauhan**
Made changes to the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,32 @@
...
+You only need basic HTML, CSS skills for getting started
 
-Several options are available to get started quickly:
-* [Download the latest release on Start Bootstrap](http://startbootstrap.com/template-overviews/agency/)
-* Clone the repo: `git clone https://github.com/BlackrockDigital/startbootstrap-agency.git`
-* Fork the repo
+If you want to contribute to the website :
 
-## Developing Using Source Files
+* Download the [ZIP FILE](https://github.com/fcc-hyd/fcc-hyd.github.io/archive/master.zip).
```
Comment (vramana) [inline README.md]: Why download the ZIP file? 😕

File: LICENSE [hunk 2]
```diff
@@ -1,8 +1,6 @@
 
 The MIT License (MIT)
 
-Copyright (c) 2013-2016 Blackrock Digital LLC.
```
Comment (vramana) [inline LICENSE]: This line should be `Copyright (c) FCC Hyderabad Community`

File: README.md [hunk 3]
```diff
@@ -1,37 +1,32 @@
-# [Start Bootstrap](http://startbootstrap.com/) - [Agency](http://startbootstrap.com/template-overviews/agency/)
+# [FCC-HYD](https:/fcc-hyd.github.io)
 
-[Agency](http://startbootstrap.com/template-overviews/agency/) is a one page agency portfolio theme for [Bootstrap](http://getbootstrap.com/) created by [Start Bootstrap](http://startbootstrap.com/). This theme features several content sections, a responsive portfolio grid with hover effects, full page portfolio item modals, a responsive timeline, and a working PHP contact form.
+This is the repo for community website created by the FCC-Hyderabad community , for the FCC-Hyderabad community.
 
 ## Getting Started
```
Comment (vramana) [inline README.md]: Probably this should be `## Contributions`

File: README.md [hunk 4]
```diff
@@ -1,37 +1,32 @@
...
+You only need basic HTML, CSS skills for getting started
```
Comment (vramana) [inline README.md]: Probably you can add  `If you are first time contributor, look at issues with Easy-to-fix label`

**Comment by himanshuchauhan** [comment]
Resolved issues according to comments

File: README.md [hunk 5]
```diff
@@ -1,37 +1,33 @@
...
+This is the repo for community website created by the FCC-Hyderabad community , for the FCC-Hyderabad community.
 
-## Getting Started
+## Contributions    
 
-Several options are available to get started quickly:
-* [Download the latest release on Start Bootstrap](http://startbootstrap.com/template-overviews/agency/)
-* Clone the repo: `git clone https://github.com/BlackrockDigital/startbootstrap-agency.git`
-* Fork the repo
+You only need basic HTML, CSS skills for getting started
```
Comment (vramana) [inline README.md]: Do we need this line? If you intend to keep it, add a period at the end.

Comment (himanshuchauhan) [inline README.md]: Removed it

File: README.md [hunk 6]
```diff
@@ -1,37 +1,32 @@
...
-* Clone the repo: `git clone https://github.com/BlackrockDigital/startbootstrap-agency.git`
-* Fork the repo
 
-## Developing Using Source Files
+If you want to contribute to the website :
 
-To use the source files, you will need to have npm installed globally along with Gulp.js. To start:
-* Run `npm install` in the root directory
-* Run `gulp dev` and edit the files as needed
+* Clone the repo: `git clone https://github.com/fcc-hyd/fcc-hyd.github.io.git` .
```
Comment (vramana) [inline README.md]: Unnecessary period here

File: README.md [hunk 7]
```diff
@@ -1,37 +1,32 @@
...
+* Fork the repo.
```
Comment (vramana) [inline README.md]: Same here

**Comment by vramana** [comment]
@himanshuchauhan You are rebasing everytime or using `git commit --amend` ?

**Comment by himanshuchauhan** [comment]
rebaseing everytime . which should I use.

**Comment by vramana** [comment]
`git commit --ammend` is a little less work compare to `git rebase` I feel. You can use whatever feels comfortable to you.

**Pull request merged**